lib/ES/Util.pm

```diff
@@ -92,7 +95,8 @@ sub build_chunked {
                 # missing attributes!
                 # '-a' => 'attribute-missing=warn',
                 '-a' => 'asciidoc-dir=' . $asciidoc_dir,
-                $resources ? ( '-a' => 'resources=' . join(',', @$resources)) : (),
+                '-a' => 'resources=' . join(',', @$resources),
+                '-a' => 'copy-callout-images=png',
```

Comment:
Does this mean they'll always and forever have to be png? That may be totally fine, I just am asking to be sure that it is known and definitely totally fine :)

Reply:
They are current `png`s. We could change it here to whatever we have images for though.